.travis.yml

```diff
@@ -41,8 +41,10 @@ env:
 
 install:
 - testing/install_test_dependencies.sh
-- pecl install mongodb || /bin/true
-- phpenv config-add testing/mongodb.ini
+- if [ ! `php -r 'echo extension_loaded("mongodb");'` ]; then
+  pecl install mongodb || /bin/true;
```

Comment:
Thanks, in this case, if we failed to install mongodb, the test can fail.
Can you remove `|| /bin/true` part?